<a href="https://colab.research.google.com/github/gilesitorr/Estadistica_Avanzada/blob/main/Tarea_2/EA_tarea_2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# __Estadística Avanzada__
# Tarea 2
## _Gil Estéfano Rodríguez Rivera_

## Primera parte

__For	each	of	the	following	exercises	construct	90,	95,	and	99	percent	confidence	intervals	for	the	population	mean,	 and	state	the	practical	and	probabilistic	interpretations	of	each.__

__Indicate	which	interpretation	you	think	would	be	 more	appropriate	to	use	when	discussing	confidence	intervals	with someone	who	has	not	had	a	course	in	statistics,	 and	state	the	reason	for	your	choice.	Explain	why	the	three	intervals	that	you	construct	are	not	of	equal	width.__

__Indicate	which	of	the	three	intervals	you	would	prefer	to	use	as	an	estimate	of	the	population	mean,	and	state	the	reason	for	your	choice.__

__En respuesta al segundo inciso__: El intervalo de confianza es un rango de valores, este rango se define a partir de la renuencia a _equivocarse_ al rechazar la hipótesis inicial. Entre más _confianza_ (o fe) le tengas a la hipótesis inicial, menos vas a querer rechazarla y vas a requerir de información muy contundente para que cambies de opinión y rechaces esa hipótesis.

__En respuesta al tercer inciso__: El valor más usado en general es la confianza del 95%, por ello, salvo que quiera estar muy seguro de mi afirmación y requiera de información más contundente (en tal caso usaría uno de 99% o más).



### Funciones útiles

Se definen algunas funciones que servirán para armar los intervalos de confianza (Si bien existen funciones que lo hacen automáticamente en R, al ser una de las partes que se evalúan en esta parte del curso, se optó por hacer las cosas "a mano").

In [1]:
#Antes, se define la función para calcular los intervalos de confianza de
#distribuciones normales
normal.confidence.intervals <- function(mean, std){

  #Vectores de los intervalos
  #Se definen con una secuencia por el tamaño de datos que se introduciran, pero
  #los valores de cada entrada son irrelevantes
  confidence.up <- 1:3
  confidence.down <- 1:3

  #Se hacen los cálculos para cada valor de confianza
  j=1

  for (i in c(90,95,99)){
    alpha <- 1-i/100
    z.value <- qnorm(1-alpha/2, mean = 0, sd = 1)
    confidence.up[j] <- mean + z.value*std
    confidence.down[j] <- mean - z.value*std
    j=j+1
  }

  #Se regresa una lista con cada uno de los valores de los intervalos
  return.list <- list("up"=confidence.up, "down"=confidence.down)

  return( return.list )
}

In [2]:
#Antes, se define la función para calcular los intervalos de confianza de
#distribuciones Student con varianzas diferentes
student.confidence.intervals <- function(mean.1, mean.2, std.1, std.2, n.1, n.2){

  mean <- mean.1-mean.2

  w.1 <- (std.1**2)/n.1
  w.2 <- (std.2**2)/n.2
  deviation <- sqrt(w.1+w.2)

  #Vectores de los intervalos
  #Se definen con una secuencia por el tamaño de datos que se introduciran, pero
  #los valores de cada entrada son irrelevantes
  confidence.up <- 1:3
  confidence.down <- 1:3

  #Se hacen los cálculos para cada valor de confianza
  j=1

  for (i in c(90,95,99)){
    alpha <- 1-i/100

    #El t-value de la distribución de cada muestra
    t.1 <- qt(1-alpha/2, n.1-1)
    t.2 <- qt(1-alpha/2, n.2-1)

    #El t-value de la prueba con varianzas diferentes
    t.prime <- (w.1*t.1 + w.2*t.2)/(w.1+w.2)

    confidence.up[j] <- mean + t.prime*deviation
    confidence.down[j] <- mean - t.prime*deviation
    j=j+1
  }

  #Se regresa una lista con cada uno de los valores de los intervalos
  return.list <- list("up"=confidence.up, "down"=confidence.down)

  return( return.list )
}

In [3]:
#Antes, se define la función para calcular los intervalos de confianza de
#distribuciones Student con varianzas iguales
student.confidence.intervals.same.variance <- function(mean.1, mean.2, std.1, std.2, n.1, n.2){

  mean <- mean.1-mean.2

  #Se calcula s_p
  sp <- (n.1-1)*(std.1**2) + (n.2-1)*(std.2**2)
  sp <- sqrt( sp/(n.1+n.2-2) )

  #Se calcula el error estándar de la distribución
  deviation <- sqrt( (sp**2)/n.1 + (sp**2)/n.2 )

  #Vectores de los intervalos
  #Se definen con una secuencia por el tamaño de datos que se introduciran, pero
  #los valores de cada entrada son irrelevantes
  confidence.up <- 1:3
  confidence.down <- 1:3

  #Se hacen los cálculos para cada valor de confianza
  j=1

  for (i in c(90,95,99)){
    alpha <- 1-i/100

    #El t-value para la prueba con varianzas iguales
    t <- qt(1-alpha/2, n.1+n.2-1)

    confidence.up[j] <- mean + t*deviation
    confidence.down[j] <- mean - t*deviation
    j=j+1
  }

  #Se regresa una lista con cada uno de los valores de los intervalos
  return.list <- list("up"=confidence.up, "down"=confidence.down)

  return( return.list )
}

### 6.2.5


__Some studies of Alzheimer’s disease (AD) have shown an increase in 14CO2 production in patients with the disease. In one such study the following 14CO2 values were obtained from 16 neocortical biopsy samples from AD patients.__

1009 1280 1180 1255 1547 2352 1956 1080

1776 1767 1680 2050 1452 2857 3100 1621

__Assume that the population of such values is normally distributed with a standard deviation of 350__

Se usará una distribución normal por indicación del problema.

In [4]:
#Se define la muestra a estudiar
data <- c(1009, 1280, 1180, 1255, 1547, 2352, 1956, 1080,
                1776, 1767, 1680, 2050, 1452, 2857, 3100, 1621)

#La desviación dada por el problema
std <- 350

#Se calcula el promedio de la muestra
mean <- mean(data)
#Se calcula el tamaño de la muestra
n <- length(data)

#Se calcula el error estándar respecto a la media
std.error <- std/sqrt(n)

In [5]:
#Calcula los intervalos
confidence <- normal.confidence.intervals(mean, std.error)

confidence.up <- confidence$up
confidence.down <- confidence$down

Como referencia, el promedio

In [6]:
mean

[1] 1747.625

Las primeras componentes de cada vector corresponden a los intervalos del 90%, la segunda al de 95% y la tercera al de 99%.

In [7]:
confidence.up

[1] 1891.550 1919.122 1973.010

In [8]:
confidence.down

[1] 1603.700 1576.128 1522.240

### 6.3.5


__A sample of 16 ten-year-old girls had a mean weight of 71.5 and a standard deviation of 12 pounds, respectively.__

__Assuming normality, find the 90, 95, and 99 percent confidence intervals for m.__

Se usará una distribución normal por indicación del problema.

In [9]:
#La desviación dada por el problema
std <- 12

#El promedio dado por el problema
mean <- 71.5

#El tamaño de la muestra
n <- 16

#Se calcula el error estándar
std.error <- std/sqrt(n)

In [10]:
#Calcula los intervalos
confidence <- normal.confidence.intervals(mean, std.error)

confidence.up <- confidence$up
confidence.down <- confidence$down

Las primeras componentes de cada vector corresponden a los intervalos del 90%, la segunda al de 95% y la tercera al de 99%.

In [11]:
confidence.up

[1] 76.43456 77.37989 79.22749

In [12]:
confidence.down

[1] 66.56544 65.62011 63.77251

### 6.4.5


__Krantz et al. (A-12) investigated dose-related effects of methadone in subjects with torsade de pointes a polymorphic ventricular tachycardia.__

__In the study of 17 subjects, nine were being treated with methadone for opiate dependency and eight for chronic pain.__

__The mean daily dose of methadone in the opiate dependency group was 541 mg/day with a standard deviation of 156, while the chronic pain group received a mean dose of 269 mg/day with a standard deviation of 316.__

Se usará una distribución t porque el tamaño de las muestras es muy pequeño (menor a 30).

En la prueba de hipótesis, se hará la suposición de que las varianzas de las dos muestras son diferentes. Esto es porque aunque ambos grupos son suministrados con la misma sustancia, sus condiciones y, por ende, sus posibles reacciones sean diferentes.

In [13]:
#La desviación dada por el problema
std.1 <- 156
std.2 <- 316

#El promedio dado por el problema
mean.1 <- 541
mean.2 <- 269

#Número de cada muestra
n.1 <- 9
n.2 <- 8


Se hará una prueba para determinar si los tratamientos tienen resultados diferentes o no.

In [14]:
#Calcula los intervalos
confidence <- student.confidence.intervals(mean.1, mean.2, std.1, std.2, n.1, n.2)

confidence.up <- confidence$up
confidence.down <- confidence$down

La diferencia de los promedios como referencia

In [15]:
mean.1-mean.2

[1] 272

Las primeras componentes de cada vector corresponden a los intervalos del 90%, la segunda al de 95% y la tercera al de 99%.

In [16]:
confidence.up

[1] 504.7031 562.1099 700.0847

In [17]:
confidence.down

[1]   39.29691  -18.10990 -156.08472

Los intervalos de confianza de 95% y 99% contienen el 0, por lo que en ese caso, se puede inferir que los tratamientos son iguales.

#### __Extra__: Si se usa una prueba pero suponiendo varianzas iguales

In [18]:
#Calcula los intervalos
confidence <- student.confidence.intervals.same.variance(mean.1, mean.2, std.1, std.2, n.1, n.2)

confidence.up <- confidence$up
confidence.down <- confidence$down

In [19]:
confidence.up

[1] 479.0708 523.4317 618.4198

In [20]:
confidence.down

[1]  64.92918  20.56828 -74.41979

En este caso, se obtienen intervalos _menos conservadores_. Es decir, desde los intervalos de confianza del 95% ya se infiere que las dosis son diferentes.

## Segunda parte


__For each of the following exercises state the practical and probabilistic interpretations of the interval that you construct.__

__Identify each component of the interval: point estimate, reliability coefficient, and standard error. Explain why the reliability coefficients are not the same for all exercises__

### Funciones útiles

In [21]:
#Antes, se define la función para calcular el intervalo de confianza del 95%
#para una distribución binomial (distribución de proporciones)
proportion.confidence.95 <- function(mean, var, n){

  #Se hacen los cálculos para el valor de confianza de 95%
  alpha <- 1-95/100
  z.value <- qnorm(1-alpha/2, mean = 0, sd = 1)

  confidence.up <- mean + z.value*sqrt(var/n)
  confidence.down <- mean - z.value*sqrt(var/n)

  #Se regresa una lista con cada uno de los valores de los intervalos
  return.list <- list("up"=confidence.up, "down"=confidence.down)

  return( return.list )
}

### 6.5.1
__Luna et al. (A-14) studied patients who were mechanically ventilated in the intensive care unit of six hospitals in Buenos Aires, Argentina.__

__The researchers found that of 472 mechanically ventilated patients, 63 had clinical evidence of ventilator-associated pneumonia (VAP).__

__Construct a 95 percent confidence interval for the proportion of all mechanically ventilated patients at these hospitals who may be expected to develop VAP__

Se usará una distribución binomial, pues se trata de un problema de lidiar con proporciones.

In [22]:
#Se calcula la proporción de la muestra
mean <- 63/472

#La varianza dada por el problema
var <- mean*(1-mean)

#El tamaño de la muestra
n <- 472

In [23]:
#Calcula los intervalos
confidence <- proportion.confidence.95(mean, var, n)

confidence.up <- confidence$up
confidence.down <- confidence$down

Como referencia, la proporción

In [24]:
mean

[1] 0.1334746

Los intervalos del 95%

In [25]:
confidence.up

[1] 0.1641554

In [26]:
confidence.down

[1] 0.1027938

## Tercera parte


__For each of the following exercises state the practical and probabilistic interpretations of the interval that you construct.__

__Identify each component of the interval: point estimate, reliability coefficient, and standard error. Explain why the reliability coefficients are not the same for all exercises__

### Funciones útiles

In [27]:
#Antes, se define la función para calcular el intervalo de confianza del 95%
#para una distribución binomial (distribución de proporciones) donde se 
#estudie la diferencia de dos muestras
proportion.difference.95 <- function(mean.1, mean.2, var.1, var.2, n.1, n.2){

  #Se hacen los cálculos para el valor de confianza de 95%
  alpha <- 1-95/100
  z.value <- qnorm(1-alpha/2, mean = 0, sd = 1)

  #Los valores para comparar las muestras
  deviation <- sqrt(var.1/n.1 + var.2/n.2)
  mean <- mean.1-mean.2

  confidence.up <- mean + z.value*deviation
  confidence.down <- mean - z.value*deviation

  #Se regresa una lista con cada uno de los valores de los intervalos
  return.list <- list("up"=confidence.up, "down"=confidence.down)

  return( return.list )
}

### 6.6.1


__Horwitz et al. (A-18) studied 637 persons who were identified by court records from 1967 to 1971 as having experienced abuse or neglect. For a control group, they located 510 subjects who as children attended the same elementary school and lived within a five-block radius of those in the abused/neglected group.__

__In the abused/neglected group, and control group, 114 and 57 subjects,
respectively, had developed antisocial personality disorders over their lifetimes.__

__Construct a 95 percent confidence interval for the difference between the proportions of subjects developing antisocial personality disorders one might expect to find in the populations of subjects from which the subjects of this study may be presumed to have been drawn__

Se usará una distribución binomial, pues se trata de un problema de lidiar con proporciones.

In [28]:
#Se calcula la proporción de la muestra
mean.1 <- 114/637
mean.2 <- 57/510

#La varianza dada por el problema
var.1 <- mean.1*(1-mean.1)
var.2 <- mean.2*(1-mean.2)

#El tamaño de la muestra
n.1 <- 637
n.2 <- 510

In [29]:
#Calcula los intervalos
confidence <- proportion.difference.95(mean.1, mean.2, var.1, var.2, n.1, n.2)

confidence.up <- confidence$up
confidence.down <- confidence$down

Como referencia, la proporción

In [30]:
mean.1-mean.2

[1] 0.06719919

Los intervalos del 95%

In [31]:
confidence.up

[1] 0.1076202

In [32]:
confidence.down

[1] 0.02677819

### 6.7.3


__A physician would like to know the mean fasting blood glucose value (miligrams per 100ml) of patients seen in a diabetes clinic over the past 10 years.__

__Determine the number of records the physician should examine in order to obtain a 90 percent confidence interval for mu if the desired width of the interval is 6 units and a pilot sample yields a variance of 60.__

In [33]:
var <- 60 #Varianza medida

alpha <- 1-90/100 #Significancia deseada
z.value <- qnorm(1-alpha/2, mean = 0, sd = 1) #Valor de z_{1-\alpha/2}

Dado que se nos proporciona un tamaño deseado para el intervalo de confianza, de éste se puede inferir el error estándar que se desea.

In [34]:
R <- 6 #Tamaño del intervalo de confianza

#La definición del intervalo de confianza:
# intervalo = estadístico +/- (Z score)*(error estándar)

#El tamaño del intervalo de confianza es de:
# R = 2*(Z score)*(error estándar).

#Al despejar, se obtiene que el error estándar dado el
#tamaño del intervalo es de:
# error estándar = R / ( 2*(Z score) )

standard.error <- R/(2*z.value) #Error estándar deseado

In [35]:
#La d de Cohen
d <- z.value * standard.error

In [36]:
#El tamaño de la muestra
n <- (z.value**2)*var/(d**2)
n

[1] 18.03696

El tamaño estimado de la muestra es de 18 personas.

### 6.8.3


__A hospital administrator wishes to know what proportion of discharged patients is unhappy with the care received during hospitalization.__

__How large a sample should be drawn if we let d=0.05, the confidence coefficient is .95, and no other information is available?__

__How large should the sample be if p is approximated by .25?__

In [37]:
alpha <- 1-95/100 #Significancia deseada
z.value <- qnorm(1-alpha/2, mean = 0, sd = 1) #Valor de z_{1-\alpha/2}

d <- 0.05 #La d de Cohen

No es posible determinar el tamaño de la distribución si no se conoce la varianza de la muestra (o alguna cantidad relacionada a ésta). Sin embargo, para una distribución binomial (de proporciones), el máximo del tamaño mínimo de la muestra se da cuando p=0.5.
¿Cuál es ese valor?

In [38]:
#Sea p=0.5
p <- 0.5
var <- p*(1-p)

#El tamaño de la muestra
n <- (z.value**2)*var/(d**2)
n

[1] 384.1459

En el caso máximo (p=0.5), se requiere de una muestra de 384 personas.

In [39]:
#Si se define p
p <- 0.25
var <- p*(1-p)

#El tamaño de la muestra
n <- (z.value**2)*var/(d**2)
n

[1] 288.1094

Si p=0.25, entonces el tamaño estimado de la muestra es de 288 personas.

### 6.9.7

__Twenty air samples taken at the same site over a period of 6 months showed the following amounts of suspendes particulate matter (micrograms per cubic meter of air):__

68 22 36 32

42 24 28 38

30 44 28 27

28 43 45 50

79 74 57 21

__Consider these measurements to be a random sample from a population of normally distributed measurements, and construct a 95 percent confidence interval for the population variance.__

In [40]:
data <- c(68, 22, 36, 32,
          42, 24, 28, 38,
          30, 44, 28, 27,
          28, 43, 45, 50,
          79, 74, 57, 21) #Muestra

n <- length(data) #Tamaño de la muestra
std <- sd(data) #Desviación de la muestra

Como se está estudiando la desviación de la distribución, se usará una prubea chi cuadrada.

In [41]:
alpha <- 1-95/100 #Significancia deseada
chi.value.down <- qchisq(1-alpha/2, df= n-1) #Valor de \chi^2_{1-\alpha/2}
chi.value.up <- qchisq(alpha/2, df= n-1) #Valor de \chi^2_{\alpha/2}

In [42]:
confidence.up <- sqrt( (n-1)/chi.value.up ) * std #Cota superior de la desviación estándar
confidence.down <- sqrt( (n-1)/chi.value.down ) * std #Cota inferior de la desviación estándar

Como referencia, la varianza de la muestra

In [43]:
std**2

[1] 295.6421

Los intervalos de confianza del 95% la desviación poblacional

In [44]:
confidence.up**2 #Cota superior

[1] 630.6843

In [45]:
confidence.down**2 #Cota inferior

[1] 170.9833

### 6.10.7

__Measurements of gastric secretion of hydrochloric acid (milliequivalents per hour) in 16 normal subjects and 10 subjects with duodenal ulcer yielded the following results:__

Normal subjects: 6.3, 2.0, 2.3, 0.5, 1.9, 3.2, 4.1, 4.0, 6.2, 6.1, 3.5, 1.3, 1.7, 4.5, 6.3, 6.2

Ulcer subjects: 13.7, 20.6, 15.9, 28.4, 29.4, 18.4, 21.1, 3.0, 26.2, 13.0

__Construct a 95 percent confidence interval for the ratio of the two population variances.__

__What assumptions must be met for this procedure to be valid?__

In [46]:
#Los datos de las dos muestras
data.1 <- c(13.7, 20.6, 15.9, 28.4, 29.4,
            18.4, 21.1, 3.0, 26.2, 13.0) #Pacientes ulcerados

data.2 <- c(6.3, 2.0, 2.3, 0.5, 1.9, 3.2, 4.1, 4.0,
            6.2, 6.1, 3.5, 1.3, 1.7, 4.5, 6.3, 6.2) #Grupo de control
            

#El tamaño de cada muestra
n.1 <- length(data.1)
n.2 <- length(data.2)

#Las desviaiones estándar
std.1 <- sd(data.1)
std.2 <- sd(data.2)

#La proproción de las varianzas
var.proportion <- (std.1/std.2)**2

Como se estudia la proporción de las desviaciones de dos distribuciones, se usará una distribución F

In [47]:
alpha <- 1-95/100 #Significancia deseada

f.value.down <- qf(1-alpha/2, df1= n.1-1, df = n.2-1) #Valor de F_{1-\alpha/2}
f.value.up <- qf(alpha/2, df1= n.1-1, df = n.2-1) #Valor de F_{\alpha/2}

In [48]:
confidence.up <- var.proportion/f.value.up #Intervalo superior
confidence.down <- var.proportion/f.value.down #Intervalo inferior

Como referencia, la porporción de las varianzas de las muestras (la de los pacientes con úlceras entre la del grupo de control)

In [49]:
var.proportion

[1] 15.99418

Los intervalos de confianza del 95% la proporción de las varianzas

In [50]:
confidence.up #Cota superior

[1] 60.28778

In [51]:
confidence.down #Cota inferior

[1] 5.121888

## Review Questions and Exercises

### 1. What is statistical inference?

Es el proceso de deducir las características de una población a partir del estudio de una muestra (que es un subconjunto de tal población).

### 2. Why is estimation an important type of inference?

Porque es un método que permite obtener información relevante sobre una población sin tener que estudiar la totalidad de la misma (proceso que puede ser incluso imposible en el mundo real).

### 3. What is a point estimate?

Es un valor que se obtiene de la muestra y que se usa para estimar un parámetro de la población.

### 4. Explain the meaning of unbiasedness

Un valor estimador __no está sesgado__ cuando, luego de medir _muchas veces_ el mismo, se obtiene que el promedio de los estimadores es _similar_ al parámetro real de la población que se pretendía estimar.

### 5. Define the following:
(a) Reliability coefficient (b) Confidence coefficient (c) Precision

(d) Standard error (e) Estimator (f) Margin of error

__Reliability coefficient (Coeficiente de confiabilidad)__:

Es el valor de la distribución estandarizada (ya sea la z -o normal estandarizada-, la t, la chi cuadrada, la F, etc) que se asocia al coeficiente de confianza que se desea.


__Confidence coefficient (Coeficiente de confianza)__:

En el sentido práctico, es el valor entre 1 y 0 que determina cuánta certeza hay de que el parámetro real que se está estimando se encuentre dentro del intervalo de confianza. El intervalo se define como:

 _estimador_ +/- ( _coeficiente de confiabilidad_ )x( _error estándar_ )

__Precision (Precisión)__:

Es la distancia del estimador hacia uno de los extremos del intervalo de confianza:

_precisión_ = ( _coeficiente de confiabilidad_ )x( _error estándar de la media_ )

__Standard error (Error estándar)__:

Es la desviación estándar de un estimador. Es decir, es una medida de la dispersión de la muestra en torno a ese estadístico.

__Estimator (Estimador)__:

Es una regla que indica cómo estimar un parámetro. Puede haber diversos estimadores para calcular un mismo parámetro.


__Margin of error (Margen de error)__:

Es un sinónimo de la __precisión__.

### 6. Give the general formula for a confidence interval

estimador +/- ( coeficiente de confiabilidad )x( error estándar )

### 7. State the probabilistic and practical interpretations of a confidence interval.

__Probabilística__:

Luego de múltiples muestreos, la proporción del número de muestras correspondiente (usualmente el 95% y, en general, el porcentaje que se desee) va a tener intervalos de confianza que contengan el parámetro real.

__Práctica__:

La proporción que se escoja (usualmente el 95% y, en general, el porcentaje que se desee) indica la probabilidad de que el intervalo de confianza de la muestra de estudio contenga el parámetro real.

### 8. Of what use is the central limit theorem in estimation?

El Teorema del Límite Central indica que, dadas n variables aleatorias con distribuciones de probabilidad con varianzas no nulas y finitas y medias conocidas, entonces la suma de variables tiende a una distribución normal.

Este teorema es útil porque permite hacer estimaciones sobre los parámetros de la población (con un valor promedio y un error estándar) con sólo hacer diferentes muestreos.

### 9. Describe the t distribution.

Es una distribución simétrica con media 0. Tiene forma de campana y tiene como dominio los números reales. Depende de un parámetro conocido como __grados de libertad__ (que es igual al tamaño de la muestra de estudio menos uno). Tiende a una distribución normal para muestras _grandes_.

### 10. What are the assumptions underlying the use of the t distribution in estimating a single population mean?

Lo primero a considerar es el tamaño de la muestra (la distribución t se usa cuando la muestra tiene menos de 30 elementos).

Se tiene que hacer la suposición de que se conoce la varianza de la población. También se debe hacer lasuposición de que la población sigue una distribución normal.

### 11. What is the finite population correction? When can it be ignored?

El cálculo del tamaño de población más elemental parte de suponer que se tienen poblaciones _grandes_ (o infinitas, en el sentido práctico). En éstas, se supone que los muestreos sin reemplazo no suponen un traslape de datos en distintas muestras. Es decir, es un método que funciona cuando se tiene _certeza_ de que un mismo elemento no va a aparecer en dos muestras diferentes. Esta fórmula no depende del tamaño de la población (pues éste no tiene efecto al ser infinito).

Cuando no se puede asegurar lo último, es decir, cuando la muestra es _suficientemente pequeña_ (o finita), se utiliza una corrección para el cálculo de la muestra. Esa corrección depende del tamaño de la población.

### 12. What are the assumptions underlying the use of the t distribution in estimating the difference between two population means?

Se supone que ambas muestras de estudio vienen de poblaciones normalmente distribuidas. Asimismo, se puede suponer que las varianzas de ambas muestras son iguales o distintas: lo importante es que sean conocidas.